src/Bundle/DependencyInjection/FriendsOfBehatSymfonyExtensionExtension.php

```diff
@@ -24,6 +24,9 @@ public function load(array $configs, ContainerBuilder $container): void
         $container
             ->registerForAutoconfiguration(Context::class)
             ->addTag('fob.context')
+            ->setBindings([
+                '$minkParameters' => new Reference('behat.mink.parameters'),
```

Comment:
This will allow FOB/PageObjectExtension's pages and elements to be autowired & autoconfigured! 🎉